programs/pluto/connections.c

```diff
@@ -311,6 +313,11 @@ delete_connection(struct connection *c, bool relations)
     pfree(c);
 }
 
+void delete_connection(struct connection *c, bool relations)
```

Comment:
If the only purpose of renaming is so that you can set force to false, then just change the signature of the function, and do a global/search replace, and/or use a #define.

Reply:
Changed the signature of delete_connection() and updated all the callers